### PR TITLE
docs *: Add files about monitoring #169

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?

/documentation

### Which issue(s) this PR fixes?

#169 

### What this PR does?

Summary:
1. Add files about monitoring feature
2. Modify the path of deploy-files 
3. Modify README.md and README_zh.md

### Special notes for your reviewer?
n/a

### CODE REVIEW

File: docs/content/zh/monitor/apply_monitoring.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  selector:
+    matchLabels:
+      app.kubernetes.io/managed-by: mysql.radondb.com
+      app.kubernetes.io/name: mysql
+```
+
+您可以在 `charts/mysql-operator/values.yaml` 文件中配置`serviceMonitor`。
+
+- 新部署 Operator 时， `serviceMonitor.enabled` 默认为**true**，表示默认开启。
+- 已部署 Operator 的集群，需重新部署 Operator。
```
Comment (runkecheng) [inline docs/content/zh/monitor/apply_monitoring.md:49]: The deployed cluster of the **old version** of the operator needs to be redeployed.

File: docs/content/zh/monitor/apply_monitoring.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+```bash
+$ kubectl apply -f config/sample/mysql_v1alpha1_cluster.yaml
+cluster.mysql.radondb.com/sample created/configured
+```
+
+## 查看监控服务
+
+### 通过客户端查看
+
+开启监控后，将自动创建一个监控服务和 serviceMonitor。 您可以通过如下指令查看集群监控信息。
```
Comment (runkecheng) [inline docs/content/zh/monitor/apply_monitoring.md:84]: Not the serviceMonitor created after the monitoring is turned on, but when the operator is deployed

File: docs/content/zh/monitor/apply_monitoring.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+在 KubeSphere 企业空间部署的 RadonDB MySQL Operator 和集群，开启监控后，可在如下页面查看监控服务状态。
+
+- 在项目空间**应用负载**下的**服务**页面，点击 `<集群名称>-metrics `，可查看监控服务信息。
+
+   ![查看监控服务](../_images/monitor_service.png)
+
+- 在项目空间**应用负载**下的**容器组**页面，点击一个容器的名称，可查看该容器中 `metrics` 资源状态。
+
+   ![查看监控资源状态](../_images/pod_metrics.png)
```
Comment (runkecheng) [inline docs/content/zh/monitor/apply_monitoring.md:139]: Maybe can add a link to how to view the monitoring

Comment (molliezhang) [reply docs/content/zh/monitor/apply_monitoring.md:139]: Add a link about KS monitoring visualization.

File: docs/content/zh/monitor/apply_monitoring.md (hunk 2 above)
Comment (molliezhang) [reply docs/content/zh/monitor/apply_monitoring.md:84]: done

File: docs/content/zh/monitor/apply_monitoring.md (hunk 1 above)
Comment (molliezhang) [reply docs/content/zh/monitor/apply_monitoring.md:49]: done

File: docs/content/zh/monitor/apply_monitoring.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  selector:
+    matchLabels:
+      app.kubernetes.io/managed-by: mysql.radondb.com
+      app.kubernetes.io/name: mysql
+```
+
+您可以在 `charts/mysql-operator/values.yaml` 文件中配置`serviceMonitor`。
+
+- 新部署 Operator 时， `serviceMonitor.enabled` 默认为**true**，表示默认开启。
+- 已部署 Operator 2.1.0以下版本的集群，需重新部署 Operator。
```
Comment (runkecheng) [inline docs/content/zh/monitor/apply_monitoring.md:49]: Add a space after 2.1.0

File: docs/content/zh/monitor/apply_monitoring.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+### 步骤 2: 配置 metricsOpts
+
+`metricsOpts` 是 `mysql_v1alpha1_cluster.yaml` 文件中定义 RadonDB MySQL 集群监控的参数。
```
Comment (runkecheng) [inline docs/content/zh/monitor/apply_monitoring.md:53]: `metricsOpts` are parameters for RadonDB MySQL cluster monitoring defined in CRD `clusters.mysql.radondb.com`. The monitoring can be turned on by editing the sample file `mysql_v1alpha1_cluster.yaml` given

File: docs/content/zh/monitor/apply_monitoring.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+```bash
+$ kubectl apply -f config/sample/mysql_v1alpha1_cluster.yaml
+cluster.mysql.radondb.com/sample created/configured
+```
+
+## 查看监控服务
+
+### 通过客户端查看
+
+集群部署完成后，将自动创建一个监控服务和 serviceMonitor。 您可以通过如下指令查看集群监控服务信息。
```
Comment (runkecheng) [inline docs/content/zh/monitor/apply_monitoring.md:87]: The serviceMonitor is not created after the cluster deployment is completed, but is created when the operator is deployed

Comment (molliezhang) [reply docs/content/zh/monitor/apply_monitoring.md:87]: done

File: docs/content/zh/monitor/apply_monitoring.md (hunk 4 above)
Comment (molliezhang) [reply docs/content/zh/monitor/apply_monitoring.md:49]: done

File: docs/content/zh/monitor/apply_monitoring.md (hunk 5 above)
Comment (molliezhang) [reply docs/content/zh/monitor/apply_monitoring.md]: --->
`metricsOpts` 是 CRD  `clusters.mysql.radondb.com` 中定义 RadonDB MySQL 集群监控的参数，可通过配置`mysql_v1alpha1_cluster.yaml` 文件中参数值开启监控服务。

**Review by runkecheng — APPROVED** [review]
lgtm

**Comment by runkecheng** [comment]
ping @andyli029 , content has been reviewed and approved

**Review by andyli029 — COMMENTED** [review]
LGTM.

**Pull request merged**